docs/tutorials/ckeditor-to-widgy.rst

```diff
@@ -0,0 +1,110 @@
+Replacing CKEditor with Widgy
+==============================
+
+One use of Widgy is as a replacement to CKEditor. Suppose you have
+something like this::
+
+    class Post(models.Model):
+        title = models.CharField()
+        content = CKEDitorField()
+
+And a template like this:
+
+.. code-block:: html+django
+
+    <h1>{{ post.title }}</h1>
+    {{ post.content|safe }}
+
+With Widgy, we'll be able to replace those with::
+
+    from widgy.db.fields import WidgyField
+    class Post(models.Model):
+        title = models.CharField()
+        content = WidgyField(
+        root_choices=['page_builder.MainContent'],
+        site='foo.site.widgy_site',
+    )
+
+And the new template:
+
+.. code-block:: html+django
+
+    {% load widgy_tags %}
+    <h1>{{ post.title }}</h1>
+    {% render_root post 'content' %}
+
+And now we'll be able to use Widgy to edit Posts' content.
+
+Installation
+------------
+
+Install Widgy and page_builder, the Widgy page content builder:
+
+.. code-block:: html+django
+
+    pip install django-widgy[page_builder]
+
+Add Widgy and its dependencies to installed apps::
+
+    'filer',
+    'compressor',
+    'argonauts',
+    'easy_thumbnails',
+    'sorl.thumbnail'
+    'widgy',
+    # not required, but this is where all the existing content widgets are
+    'widgy.contrib.page_builder',
+
+Make yourself a Widgy site. This controls Widgy configuration. The
+defaults are fine for now. In foo/site.py::
+
+    from widgy.site import WidgySite
+
+    widgy_site = WidgySite()
+
+And tell Widgy to use it in your settings::
+
+    WIDGY_MEZZANINE_SITE = 'foo.site.widgy_site'
+
+This shouldn't be required for much longer, it's an open issue to remove
+this because we're not using Mezzanine right now. There is a small piece
+of code in page_builder that still depends on this setting though.
```

Comment:
no, but you have to set the setting?

*Rocky Meza*, Programmer
fusionbox*▝*
o: 303.952.7490

www.fusionbox.com
<http://www.fusionbox.com/?utm_source=fb%2Bemail&utm_medium=email&utm_campaign=email%2Btracking>

This information is confidential and intended solely for the use
of the individual or entity to whom it is addressed. If you have
received this email in error please notify the sender.

On Wed, Mar 25, 2015 at 4:42 PM, Antoine Catton <notifications@github.com>
wrote:

> In docs/tutorials/ckeditor-to-widgy.rst
> <https://github.com/fusionbox/django-widgy/pull/304#discussion_r27174008>:
>
> > +    'widgy.contrib.page_builder',
> > +
> > +Make yourself a Widgy site. This controls Widgy configuration. The
> > +defaults are fine for now. In foo/site.py::
> > +
> > +    from widgy.site import WidgySite
> > +
> > +    widgy_site = WidgySite()
> > +
> > +And tell Widgy to use it in your settings::
> > +
> > +    WIDGY_MEZZANINE_SITE = 'foo.site.widgy_site'
> > +
> > +This shouldn't be required for much longer, it's an open issue to remove
> > +this because we're not using Mezzanine right now. There is a small piece
> > +of code in page_builder that still depends on this setting though.
>
> does this means that we need to pip install django-widgy[widgy_mezzanine]
> as well?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/fusionbox/django-widgy/pull/304/files#r27174008>.
>